### PR TITLE
Branch Update Documentation

### PR DESCRIPTION
**Pull request by KhoonSun47**
Update User Guide and Developer Guide:

Changes made:
- UserGuide.md: Add glossary and update page break for PDF viewing
- DeveloperGuide.md: Remove commented lines
- DeveloperGuide.md: Change the width of addSeller sequence and activity diagram
- AddSellerSequenceDiagram-Logic.puml: Change the arguments so that it looks nicer when viewing the sequence diagram

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -524,6 +528,24 @@ Format: `matchBuyer FULL_NAME`
 
 <div style="page-break-after: always;"></div>
 
+[//]: # (@@author redcolorbicycle)
```
Comment (felixchanyy) [inline docs/UserGuide.md:531]: Missing ending author tag

File: docs/UserGuide.md [hunk 2]
```diff
@@ -629,3 +653,22 @@ Action     | Format, Examples
 **Clear**         | `clear`
 **Exit**          | `exit`
 **Help**          | `help`
+
+<div style="page-break-after: always;"></div>
+
+## Glossary
```
Comment (felixchanyy) [inline docs/UserGuide.md:659]: Missing Price and Budget

File: docs/UserGuide.md (hunk 1 above)
Comment (lokidoki102) [reply docs/UserGuide.md:531]: I agree, this would cause @redcolorbicycle to be the author for all the codes below.

Comment (redcolorbicycle) [inline docs/UserGuide.md:531]: Missing ending author tag

File: docs/UserGuide.md (hunk 2 above)
Comment (redcolorbicycle) [inline docs/UserGuide.md:659]: Missing price and budget

**Review by redcolorbicycle — APPROVED** [review]

**Review by lokidoki102 — APPROVED** [review]

**Pull request merged**